flex/interactive/sdk/python/gs_interactive/client/session.py

```diff
@@ -814,4 +830,13 @@ def try_upload_files(self, schema_mapping: SchemaMapping) -> Result[SchemaMappin
             return upload_res
         print("new schema_mapping: ", upload_res.get_value())
         return Result.ok(upload_res.get_value())
-        
\ No newline at end of file
+
+    def ensure_param_str(self, param_name : str, param):
```

Comment:
这个这么写好了：
```
ret = param
if not isinstance(param, str):
    try:
        ret = str(param)
    except e:
        raise  ValueException(f"{param_name} has a non-string value {param}, and cannot be converted into a string")

return ret
```

Reply:
I'm considering whether converting any object to a string is advisable in our situation. For instance, we convert an integer value to a string because it's a frequent mistake when using the Interactive SDK. Concerning other types, perhaps it's more appropriate to trigger the error on the client side instead of sending a request to the server knowing that it will fail.